CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## Version 0.7.0
+
+Under development.
+
+* Dependencies:
+  - Package dependent on ``Particle`` versions 0.10.*.
```

Comment:
```suggestion
  - Package dependent on ``Particle`` version 0.10.
```
This is making the markdown unhappy, so maybe just make this 0.10, patch versions are implied.

Reply:
Sure, no prob.